esmvaltool/utils/cmorizers/obs/cmorize_obs_Landschutzer2014.py

```diff
@@ -0,0 +1,119 @@
+"""
+# #############################################################################
+# ESMValTool CMORizer for Landschutzer2014 data
+# #############################################################################
+#
+# Tier
+#    Tier 2: other freely-available dataset.
+#
+# Source
+#    ftp://ftp.nodc.noaa.gov/nodc/archive/arc0105/0160558/1.1/data/0-data/
+#
+# Last access
+#    20190227
+#
+# Download and processing instructions
+#    Download the following file in ${RAWOBS}/Tier2/Landschutzer2014:
+#     pco2_1998-2011_ETH_SOM-FFN_CDIAC_G05.nc
+#
+#
+# Modification history
+#    20190227-A_lova_to: written.
+#
+# #############################################################################
+"""
+
+import datetime
+import logging
+import os
+
+import iris
+from cf_units import Unit
+import numpy as np
+
+from .utilities import (_add_metadata,
+                        _convert_timeunits,
+                        _fix_coords,
+                        _read_cmor_config,
+                        _roll_cube_data,
+                        _save_variable)
+
+logger = logging.getLogger(__name__)
+
+# read in CMOR configuration
+cfg = _read_cmor_config('Landschutzer2014.yml')
+proj = cfg['proj']
+timestamp = datetime.datetime.utcnow()
+timestamp_format = "%Y-%m-%d %H:%M:%S"
+now_time = timestamp.strftime(timestamp_format)
+proj['metadata_attributes']['CMORcreated'] = now_time
+VAR_TO_CMOR = cfg['VAR_TO_CMOR']
+VAR_TO_FILENAME = cfg['VAR_TO_FILENAME']
+STANDARD_NAMES = cfg['STANDARD_NAMES']
+LONG_NAMES = cfg['LONG_NAMES']
+
+ALLVARS = list(VAR_TO_CMOR.keys())
+
+
+def _fix_metadata(cube, var):
+    """Fix all aspects of metadata for different vars."""
+    logger.info("Fixing units...")
+    if var in ['fgco2', ]:
+        cube.units = Unit('kg m-2 s-1')
```

Comment:
Units should be read from the cmor tables, specifying them here is unnessary and error prone

Reply:
good point!

Reply:
in this case the unit specification is a direct consequence of the data transformation made in `_fix_data`, so probably the unit change should be moved in there to make more straightforward the set of operations. It is done to make the OBS data consistent with CMOR standard.